js/keyboard.js

```diff
@@ -0,0 +1,154 @@
+import arr from './data.js';
+import Language from './language.js';
+
+export default class Keyboard {
+	static pressCaps = false;
+
```

Comment:
a lot of empty lines, which you don't need. it's about all files. think which line you should keep and which it will be better to remove

Reply:
line requires eslint